substrate/client/transaction-pool/tests/zombienet/network-specs/asset-hub-high-pool-limit-fatp-relaychain.toml

```diff
@@ -0,0 +1,83 @@
+[settings]
```

Comment:
The file could be renamed - we don't need asset-hub in the name (assuming you'll remove parachain).

Reply:
[f2409a9](https://github.com/paritytech/polkadot-sdk/pull/7257/commits/f2409a9a22f7893d8cb1dda64bc47c9fca9872b8)